pkg/controlplane/scheduling/enqueuer.go

```diff
@@ -124,6 +124,12 @@ func (e *Enqueuer) Execute(ctx context.Context, req *cloud.ScheduleRequest) ([]t
 	}
 	// END POST-COMMIT VALIDATION
 
+	// Workaround
+	err = e.assignExecutions(commitContext, intermediateExecutions)
```

Comment:
Below we dispatch execution events. I wanted to add dispatch an "assigned" event, but to my surprise this does not seem to exist.